colearn/utils/data.py

```diff
@@ -76,7 +77,10 @@ def _download_data_from_gcloud(cloud_data_dir, local_data_dir):
     bucket_name, prefix = bucket_name.split('/', 1)
     print(f"Downloading data from google cloud: Bucket {bucket_name}, prefix {prefix}")
 
-    storage_client = storage.Client()
+    if len(os.getenv(GAUTH_ENV_VAR_NAME, "")) > 0:
```

Comment:
Why not use `if "GOOGLE_APPLICATION_CREDENTIALS" not in os.environ:`?

Reply:
in general it is a good practice to avoid magical values.
I used getenv to support the case where the env var is set but empty